contracts/axelar-gateway/src/test_gateway.rs

```diff
@@ -470,28 +459,17 @@ fn transfer_operatorship() {
 }
 
 #[test]
-#[should_panic(expected = "HostError: Error(Auth, InvalidAction)")] // Unauthorized
 fn transfer_operatorship_unauthorized() {
-    let (env, contract_id, client) = setup_env();
+    let (env, _, client) = setup_env();
     let owner = Address::generate(&env);
     let operator = Address::generate(&env);
-    let new_operator = Address::generate(&env);
-    let user = Address::generate(&env);
+    let not_owner = Address::generate(&env);
 
     initialize(&env, &client, owner, operator.clone(), 1, randint(1, 10));
 
     assert_eq!(client.operator(), operator);
-    client
-        .mock_auths(&[MockAuth {
-            address: &user,
-            invoke: &MockAuthInvoke {
-                contract: &contract_id,
-                fn_name: "transfer_operatorship",
-                args: (&new_operator,).into_val(&env),
-                sub_invokes: &[],
-            },
-        }])
-        .transfer_operatorship(&new_operator);
+
+    assert_invoke_auth!(not_owner, client.try_transfer_operatorship(&not_owner));
```

Comment:
```suggestion
    assert_invoke_auth_err!(not_owner, client.try_transfer_operatorship(&not_owner));
```
Also convert the tests where the auth is meant to be successful to a macro as well

Reply:
got it. updated.